css/styles.css

```diff
@@ -116,3 +116,259 @@
+  margin-right: 12px;
+  padding: 10px 12px 10px 12px;
+  border: 1px solid #8993a4;
+  cursor: pointer;
+}
+
+.buttons li a {
+  text-decoration: none;
+  color: #091e42;
+}
+
+.see-button li {
+  margin-top: 10px;
+  background: #ff6b00;
+  border: 0;
+  font-weight: 700;
+  font-size: 17px;
+}
+
+.see-button li a {
+  color: #fff;
+}
+
+.art-printing {
+  background-image:
+    linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%),
+    url("../img/works-img/art-printing.png");
+  background-repeat: no-repeat;
+  background-size: 100%;
+}
+
+.art-title {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 60px 0 0 0;
+}
+
+.art-text {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
+.button-transparent li {
+  background: rgba(255, 255, 255, 0.24);
+  color: #fff;
+}
+
+.button-transparent li a {
+  color: #fff;
+}
+
+.see-button-2 {
+  background: #ff6b00;
+  margin-top: 22px;
+}
+
+.see-button-2 li {
+  margin-left: 114px;
+  margin-top: 5px;
+  margin-bottom: 5px;
+}
+
+.about {
+  background: #1c1a19;
+}
+
+.about-me {
+  display: flex;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6f00;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 24px 0;
+}
+
+.about-desc {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 44px 0;
+}
+
+.about-img {
+  background-image: url("../img/about-img/about-bkgrnd.png");
+  background-position: 100% 100%;
+  background-size: 35%;
+  background-repeat: no-repeat;
+  padding-bottom: 100px;
+}
+
+.line {
+  border: 1px solid #fff;
+  margin-right: 35px;
+  margin-left: 35px;
+}
+
+.skills-heading {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  margin: 0 12px;
+  text-align: center;
+}
+
+.skills-list {
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+  margin: 44px 0;
+}
+
+.language-shape {
+  list-style-image: url('../img/skills-img/language1.png');
+}
+
+.languages li {
+  list-style: none;
+}
+
+.frameworks-shape {
+  list-style-image: url('../img/skills-img/frameworks.png');
+}
+
+.frameworks li {
+  list-style: none;
+}
+
+.skills-shape {
+  list-style-image: url('../img/skills-img/skills.png');
+}
+
+.skills li {
+  list-style: none;
+}
+
+.buttons li:hover {
+  text-decoration: none;
+}
+
+.buttons li:active {
+  box-shadow: 0 0 #444;
+  transform: translateY(4px);
```

Comment:
- [ ] Please Fix the weird behavior of the hovering here  
![image](https://user-images.githubusercontent.com/74744787/158997440-980410b6-b158-45ff-8ce9-ca1bd6e0592c.png)
hovering suppose to change the hole button color not just this part

Reply:
Done.